content/release.rst

```diff
@@ -19,6 +19,19 @@ Releasing a product usually consists of publishing the following:
 * Documentation, which includes Release Notes, Known Issues and Upgrade Steps.
 * Public announcement. Notification email or website news/blog article.
 
+Each version sent to a customer should have an unique version number and
+should be part of the release notes history.
```

Comment:
_an unique_ -> _a unique_. We have this documented at https://trac.chevah.com:10443/wiki/Development/EnglishForRomanians#indefinite-article

Also, _release notes history_ sounds redundant to me, I would simply use: _and should be mentioned in the release notes._ Note that _should_ suggests a weak requirement, if this is an absolute requirements, use _must_.